llama_index/chat_engine/condense_question.py

```diff
@@ -124,12 +125,62 @@ def chat(
         response = self._query_engine.query(condensed_question)
 
         # Record response
-        chat_history.extend(
-            [
-                ChatMessage(role=MessageRole.USER, content=message),
-                ChatMessage(role=MessageRole.ASSISTANT, content=str(response)),
-            ]
-        )
+        if isinstance(response, StreamingResponse):
```

Comment:
do we really need to raise this error? could we just do `str(response)` and go on?